### PR TITLE
[INTEL MKL] Pad+Conv fusion for bf16

### PR DESCRIPTION
**Pull request by Solaryee**
This PR enables pad+conv fusion for bf16 and related test case.

### CODE REVIEW

File: tensorflow/core/kernels/mkl_fused_ops_test.cc [hunk 1]
```diff
@@ -705,65 +743,25 @@ class FusedPadConvOpTest : public OpsTestBase {
...
 };
 
-TEST_F(FusedPadConvOpTest, PaddingConvTest) {
-  const int depth = 1;
-  const int image_width = 4;
-  const int image_height = 3;
-  const int image_batch_count = 1;
-  Tensor image(DT_FLOAT, {image_batch_count, image_height, image_width, depth});
-  test::FillValues<float>(&image, {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12});
+TYPED_TEST_CASE_P(FusedPadConvOpTest);
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_ops_test.cc:750]: This is deprecated. Please use `TYPED_TEST_SUITE_P` instead.
```suggestion
TYPED_TEST_SUITE_P(FusedPadConvOpTest);
```

File: tensorflow/core/kernels/mkl_fused_ops_test.cc [hunk 2]
```diff
@@ -705,65 +743,25 @@ class FusedPadConvOpTest : public OpsTestBase {
...
-}
-
-TEST_F(FusedPadConvOpTest, PaddingConvTestNchw) {
-  const int depth = 1;
-  const int image_width = 4;
-  const int image_height = 3;
-  const int image_batch_count = 1;
-  Tensor image(DT_FLOAT, {image_batch_count, depth, image_height, image_width});
-  test::FillValues<float>(&image, {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12});
+REGISTER_TYPED_TEST_CASE_P(FusedPadConvOpTest, PaddingConvTest,
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_ops_test.cc:756]: Also deprecated.
```suggestion
REGISTER_TYPED_TEST_SUITE_P(FusedPadConvOpTest, PaddingConvTest,
```

File: tensorflow/core/kernels/mkl_fused_ops_test.cc [hunk 3]
```diff
@@ -705,65 +743,25 @@ class FusedPadConvOpTest : public OpsTestBase {
...
-       59, 12,  0,   0,   0,   0,  0,   0,   0,   0,   0,   0});
-
-  Run<float>(DT_FLOAT, image, filter, padding, expected, "NCHW");
-}
+#ifdef ENABLE_INTEL_MKL_BFLOAT16
+using FusedPadConvDataTypes = ::testing::Types<float, bfloat16>;
+#else
+using FusedPadConvDataTypes = ::testing::Types<float>;
+#endif
+INSTANTIATE_TYPED_TEST_CASE_P(Test, FusedPadConvOpTest, FusedPadConvDataTypes);
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_fused_ops_test.cc:764]:
```suggestion
INSTANTIATE_TYPED_TEST_SUITE_P(Test, FusedPadConvOpTest, FusedPadConvDataTypes);
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR and for refactoring!

File: tensorflow/core/kernels/mkl_fused_ops_test.cc (hunk 3 above)
Comment (ShengYang1) [reply tensorflow/core/kernels/mkl_fused_ops_test.cc]: Thanks for review. All `TYPED_TEST_CASE_P` in this file have been replaced

**Review by penpornk — APPROVED** [review]
Thank you for the quick response!

**Pull request merged**